src/index.js

```diff
@@ -58,16 +59,30 @@ const unvisit = (input, schema, getDenormalizedEntity) => {
     return method(schema, input, unvisit, getDenormalizedEntity);
   }
 
+  if (input === undefined || input === null) {
```

Comment:
Sorry, my brain isn't working today... I'd rather stay away from the extra configuration, as you proposed. I'd expect the application to handle receiving `null` for an entity elsewhere than pushing more into normalizr.